### PR TITLE
feature(cards): new component

### PR DESCRIPTION
**Pull request by moog16**

### CODE REVIEW

File: packages/card/README.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+# React Card
+
+MDC React Card is a component for material design cards.
+
+## Installation
+
+```
+npm install @material/react-card
```
Comment (acdvorak) [inline packages/card/README.md]: Are we publishing this to NPM right now? Or are you just planning ahead?

Also, the package names seem inconsistent; there's `react-card` and `react-fab`, but also `material-icon-react` and `top-app-bar-react`. Which one are we going with? (I vote for `react-foo`)

File: packages/card/README.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+#### Props
+
+Prop Name | Type | Description
+--- | --- | ---
+fullBleed | n/a | Enables full bleed card actions row by removing all padding.
+
+### CardContent
+
+This component is used as the container for primary tappable content.
```
Comment (acdvorak) [inline packages/card/README.md]: If this is intended to be the primary tappable area, it should be named "primary action" for consistency with MDC and the spec.

File: packages/card/README.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+import Card, {CardContent} from '@material/react-card';
+
+<Card>
+  <CardContent>
+    <p>Content</p>
+  </CardContent>
+</Card>
+```
+
+### Card
```
Comment (acdvorak) [inline packages/card/README.md]: Shouldn't this be the first component listed in the README?

I would probably reorder these components to something like:

1. Card
2. CardPrimaryAction
3. CardMedia
4. CardActions
5. CardActionButtons
6. CardActionIcons

File: packages/card/README.md (hunk 1 above)
Comment (moog16) [reply packages/card/README.md]: Just planning ahead.

And yes this PR is inconsistent, and is fixed in #16 for `react-foo` :)

File: packages/card/README.md (hunk 3 above)
Comment (moog16) [reply packages/card/README.md]: Good idea. I was going in order of how it displayed in my filesystem.

**Pull request merged**